src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --link-hover-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  position: relative;
+  max-width: 1200px;
+  min-width: 1024px;
+  display: flex;
+  /* justify-content: space-between; */
+  justify-content: flex-end;
+  /* align-items: center; */
+  padding: 0 50px;
+}
+
+.logo {
```

Comment:
better add styles to link
<img width="1496" alt="Screenshot 2023-04-27 at 19 01 52" src="https://user-images.githubusercontent.com/53228259/234920049-22580c11-0478-4ff2-8e3b-3a50efd0f5a5.png">
<img width="1487" alt="Screenshot 2023-04-27 at 19 02 14" src="https://user-images.githubusercontent.com/53228259/234920142-e11bb04c-2553-4b7e-89b6-f9bded75f7c0.png">